### DISCUSSION TITLE
Einzelnes Controll / ganzes Widget auf Read Only stellen? (Jarvis V2)

### DISCUSSION THREAD

**Discussion by manoloxxl** [Zefau/ioBroker.jarvis#2755] [Q&A] [answered]
Hi zusammen,

ich bin bei einem Projekt leider weiterhin auf Jarvis 2 unterwegs. Ich bin noch nicht dazu gekommen umzustellen...

In diesem Projekt möchte ich gerne, idealerweise über einen Datenpunkt steuerbar, die Eingabemöglichkeit in Teilen der GUI unterbinden.

1) In einem Pop-Up habe ich 3 Anzeigewerte sowie einen Schieberegler / Level Body, mit dem man die Temperatur einstellen kann. Ich möchte gerne diesen Schieberegler sperren, wenn ein spezieller Datenpunkt einen boolean hat.
Ideal wäre ein read-only, notfalls geht auch ein hide. Aber wie baue ich das?

2) im zweiten Fall würde ich das gleiche gerne mit einem ganzen Widget machen. Leider kann man das aber nicht mal eben inaktiv stellen

Wie stelle ich das an?

Grüße

**Comment by mcuiobroker** [comment, marked as answer]
> möchte gerne diesen Schieberegler sperren, wenn ein spezieller Datenpunkt einen boolean hat.

Kann man erst mit v3.
Auch die anderen "Wünsche" gehen nicht mit v2.

Also in v2 ist da nicht viel zu machen.
Kann es mir mal anschauen, aber glaube nicht, das es was wird.

**Comment by manoloxxl** [comment]
Danke für das Feedback - investier da mal keine Zeit. Ich muss eh irgendwann mal endlich Updaten.

**Comment by manoloxxl** [comment]
Ich würde hier gerne noch mal einsteigen nach den ich nun auf V3 bin. Laut [Link](https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/styles/widgets/widget-ausblenden#widget-ausblenden) kann ich Widgets ausblenden per CSS. Ich hab das aber nicht in Kombination mit einem DP hinbekommen.

Geht das in so einer Kombination auch?

.jarvis-widget-4acb0466-f47c-4eff-a1a6-22c836cabc7f {
   {"{0_userdata.0.testwert}=0":{"display":"none !important"}}
}

Ich hab es nicht geschafft. 

Die Alternative wäre dann bei jedem Gerät pro DB bei Body Stil (Gerät) immer den identischen Eintrag zu setzen? Also...
{"{0_userdata.0.testwert}=0":{"display":"none !important"}}

?

**Reply by mcuiobroker** [reply]
> Geht das in so einer Kombination auch?

> .jarvis-widget-4acb0466-f47c-4eff-a1a6-22c836cabc7f {
{"{0_userdata.0.testwert}=0":{"display":"none !important"}}
}

Nein, nur so. -> Widget wird nicht angezeigt.

```
.jarvis-widget-4acb0466-f47c-4eff-a1a6-22c836cabc7f {
    display:none !important;
}

Body-Stil ist doch viel einfach, wenn man es nicht dauerhaft ausblenden möchte?

**Reply by manoloxxl** [reply]
Ich hätte halt lieber den ganzen Kasten weg wenn ein dp gesetzt ist, aber so gehts ja auch. Ich setze mich mal ran.

Danke!
